components/writers/src/main/java/org/datacleaner/extension/output/SortHelper.java

```diff
@@ -64,10 +64,12 @@ public int compare(Row row1, Row row2) {
                     for (int i = 0; i < numberOfSelectItems; i++) {
                         final String rowValue1 = (String) row1.getValue(i);
                         final String rowValue2 = (String) row2.getValue(i);
-                        if (CompareUtils.compare(rowValue1, rowValue2) == 0) {
+                        final String rowValue1LowerCased = rowValue1.toLowerCase();
+                        final String rowValue2LowerCased = rowValue2.toLowerCase();
+                        if (CompareUtils.compare(rowValue1LowerCased, rowValue2LowerCased) == 0) {
```

Comment:
If you know they are that and they are Strings, you can just use String.compareToIgnoreCase() instead of lowercasing them.